core/trino-spi/src/main/java/io/trino/spi/function/FunctionProvider.java

```diff
@@ -13,14 +13,12 @@
  */
 package io.trino.spi.function;
 
-import io.trino.spi.Experimental;
 import io.trino.spi.function.table.ConnectorTableFunctionHandle;
 import io.trino.spi.function.table.TableFunctionProcessorProvider;
 
 import java.util.Collection;
 import java.util.List;
 
-@Experimental(eta = "2023-03-31")
```

Comment:
> Remove experimental flag from FunctionProvider
> 
> eta for experimental was 2023-03-31

eta is "estimated date".  from javadoc

> When the community expects the SPI will be finalized, but this date may be extended.

the fact that we're past ETA is no direct relation with SPI maturity and cannot be taken as the basis for removing `@Experimental`.
Please update the commit message with real reasoning supporting graduating this functionality.
I would assume you've also checked this with @dain

Reply:
sure, not touching this flag now